src/css/components/icon.scss

```diff
@@ -61,8 +82,32 @@
     fill: $color-error;
   }
 
-  &.icon-alt {
-    fill: $color-primary-alt-light;
+  &.icon-active {
+    fill: $color-blue;
+  }
+}
+
+.icon-chevron {
```

Comment:
So in the pattern library, this is called `icon-ui`. Can we keep the naming conventions consistent? I'd prefer this to be `icon-ui`, but I'll leave it to you what you think it should be based on how you think it will be used.

If it's a separate component in the styleguide, it also might make sense to be it's own component file.